MANIFEST.in

```diff
@@ -12,15 +12,13 @@ include *.yaml
 include *.yml
 include changelog/README.rst
 include CITATION.cff
-include plasmapy/py.typed
```

Comment:
This line is probably not necessary because everything within `src` is now included.